### DISCUSSION TITLE
Help us finish off OpenCV-C

### DISCUSSION THREAD

**Discussion by friolator** [friolator/OpenCV-C#1] [Announcements]
This project began as a way to use OpenCV within development environments like Xojo, which require a C-based external library, not C++. Most of the work you see here as of today was paid for by Gamma Ray Digital. There is still work to do though!

Many of OpenCV's Overloaded functions need to be ported over to OpenCV-C, and given unique names. (See CVCImgProc.cpp for examples, such as CVCintegral() and CVCintegral2() ). Additionally, we need to hand off control of this project to someone more familiar with managing an open source project. We got this started, but don't really have the time to devote to it that it would really require. If you're interested in helping to maintain the project, let's discuss!

**Comment by Michel91dev** [comment]
I would appreciate discussing the subject with you. Please let me know

**Comment by ldci** [comment]
Hi Michel
When you want for a discussion :)

[image001.png][image002.png]

François Jouen
<https://univ-psl.fr/>
École Pratique des Hautes Études
Doyen Honoraire de la section des Sciences de la Vie et de la Terre
Chaire de Psychobiologie du Développement
Plateforme Réanimation Pédiatrique Raymond Poincaré. Hôpital de Garches
Laboratoire Cognitions Humaine et Artificielle
Campus Condorcet
14 cours des Humanités
93322 Aubervilliers cedex
06 87 13 76 81
***@***.******@***.***>
www.ephe.fr<http://www.ephe.fr/>


Le 18 mars 2023 à 16:11, Michel Safars ***@***.***> a écrit :



I would appreciate discussing the subject with you. Please let me know

**Comment by friolator** [comment]
Anyone is welcome to contribute to the project. If you make changes, we can integrate them into the main branch. As stated above, the main thing that still needs to be done is dealing with all the C++ overloaded functions. If you're able to do that, by all means have at it!

**Comment by Michel91dev** [comment]
Thanks to the help of [Greg O](https://forum.xojo.com/u/greg_o/summary) and the nice discussion with [François Jouen](https://github.com/ldci), I recompiled the Library for both (Intel **and ARM**), so I could make it run on new Macs.

Here is the INTEL-ARM version: [libOpenCVC.dylib.zip](https://github.com/friolator/OpenCV-C/files/11131688/libOpenCVC.dylib.zip)
